rwengine/src/audio/SoundManager.cpp

```diff
@@ -16,6 +18,9 @@ extern "C" {
 #define av_frame_free   avcodec_free_frame
 #endif
 
+constexpr int kNumOutputChannels = 2;
```

Comment:
Why this is now hard-coded?

Reply:
Mono sound you will hear only from one channel. It will sound weird in binaural mode. Const var is to make clear that it's converted into stereo.

Reply:
Yes all of your reply makes sense.

But does this mean that GTA sound files doesn't have more than 2 channels? Did GTA3 (or VC/SA) support surround sound?

Reply:
I'm not familiar with VC/SA, but wav files and sfx are mono. Mp3 files are stereo.

Reply:
Thanks, I don't have issues with this then.

Reply:
If you think about input data, it will (or should) be converted to stereo.
If output data, due to bad quality source it'll sound as normal stereo.